test/unit/vdom/properties.js

```diff
@@ -38,13 +38,21 @@ describe('vdom/properties', () => {
 
   it('false should remove the attribute', done => {
```

Comment:
We never handled those before. The reasoning was to try and make as little bespoke behaviour on top of HTML as possible. Meaning `setAttribute('attr', false)` will set `attr="false"` (stringify). It does the same will `null` and `undefined`. Changing that so that `false` would remove the attribute (which is what was originally done here and this is a bug) made it so that we keep the bespoke footprint small. Technically adding `null` / `undefined` would be a breaking change.

We can do this, but I'd rather do it as a separate issue to make the change deliberate if we think that's a good idea. Thoughts?